docs/reference/method/MongoDBClient-createClientEncryption.txt

```diff
@@ -0,0 +1,50 @@
+=========================================
+MongoDB\\Client::createClientEncryption()
+=========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Client::createClientEncryption()
+
+   Returns a ClientEncryption object for manual encryption and decryption of
+   values.
+
+   .. code-block:: php
+
+      function createClientEncryption(array $options): MongoDB\Driver\ClientEncryption
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBClient-method-createClientEncryption-param.rst
+
+   The ``$options`` parameter supports all options documented in the extension
+   manual. For the ``keyVaultClient`` option, an instance of ``MongoDB\Client``
```

Comment:
```
:phpclass:`MongoDB\\Client`
```

Should "extension manual" link to the PHP.net docs, as you did in #708 for the `autoEncryption` driver option? I realize https://www.php.net/manual/en/mongodb-driver-manager.createclientencryption.php does not yet exist, but I suppose you can still add this now since you're linking to the same URL in the "See Also" below.